### PR TITLE
Another test inject helper

### PR DESCRIPTION
**Pull request by nerdrew**

### CODE REVIEW

File: lib/interjectable/rspec.rb [hunk 1]
```diff
@@ -107,21 +114,27 @@ def test_inject(dependency, &setter)
...
+
+      ClassMethods.test_inject(self.class, target, dependency, value)
+    end
+  end
+end
+
+if defined?(RSpec)
+  RSpec.configure do |c|
+    c.include(Interjectable::RSpecHelper)
+  end
```
Comment (zachmargolis) [inline lib/interjectable/rspec.rb]:
```suggestion
  end
else
  warn "RSpec helper was required but RSpec has not beed defined"
```

What if we added a warning for this case, like this?

Comment (nerdrew) [reply lib/interjectable/rspec.rb]: I like it. Adding. `warn` only or should I abort?

Comment (zachmargolis) [reply lib/interjectable/rspec.rb]: I don't know what's expected of a helper like this....I agree that it seems like an error, so let's error? Or maybe throw? I trust you on this.

Comment (nerdrew) [reply lib/interjectable/rspec.rb]: I think throwing is better in this case.

**Pull request merged**